litellm/tests/test_completion.py

```diff
@@ -141,7 +141,7 @@ def test_completion_openai_with_functions():
 
 def test_completion_azure():
     try:
-        response = completion(model="gpt-3.5-turbo", deployment_id="chatgpt-test", messages=messages, azure=True)
+        response = completion(model="gpt-3.5-turbo", deployment_id="chatgpt-test", messages=messages, custom_llm_provider="azure")
         # Add any assertions here to check the response
```

Comment:
not blocking, but now that we use deployment_id - do we even need azure=True ? 

Can address in separate issue